### PR TITLE
Remove fees slow, optimal and fast only in simple view

### LINKED ISSUE #8516: Remove fees slow, optimal and fast only in simple view
### Description

 Remove fees slow, optimal and fast only in simple view 
 remove the tabs in advanced view, only keep fee details

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #8516 

### What does the PR do

Remove option to switch between slow, optimal and fast priority from SendModal

### Affected areas

Wallet

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [] I've checked the design and this PR matches it
- doesn't match the design as of now, but it will be updated soon

https://user-images.githubusercontent.com/60327365/204805318-490ed8e9-5e61-4fc8-a062-f58d529fbd58.mov

### CODE REVIEW

File: ui/imports/shared/popups/SendModal.qml [hunk 1]
```diff
@@ -505,8 +505,8 @@ StatusDialog {
 
     footer: SendModalFooter {
         nextButtonText: popup.isBridgeTx ? qsTr("Bridge") : qsTr("Send")
-        maxFiatFees: popup.isLoading ? "..." : fees.selectedGasFiatValue
-        selectedTimeEstimate: popup.isLoading? "..." : fees.selectedTimeEstimate
+        maxFiatFees: popup.isLoading ? "..." : "%1 %2".arg(LocaleUtils.numberToLocaleString(d.totalFeesInFiat)).arg(popup.store.currentCurrency.toUpperCase())
```
Comment (caybro) [inline ui/imports/shared/popups/SendModal.qml:508]:
```suggestion
        maxFiatFees: popup.isLoading ? "..." : d.totalFeesInFiat.toLocaleCurrencyString(Qt.locale(), popup.store.currentCurrency))
```

File: ui/imports/shared/popups/SendModal.qml [hunk 2]
```diff
@@ -105,6 +103,9 @@ StatusDialog {
         property var preferredChainIds: []
         property bool sendTxError: false
         property string sendTxErrorString
+        property string totalTimeEstimate
+        property string totalFeesInEth
+        property string totalFeesInFiat
```
Comment (caybro) [inline ui/imports/shared/popups/SendModal.qml:108]: Why are we storing these as strings when these are clearly numbers? It prevents us from doing proper formatting :/

File: ui/imports/shared/popups/SendModal.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/popups/SendModal.qml:508]: If it were a proper Number... :/

File: src/backend/eth.nim [hunk 3]
```diff
@@ -28,6 +28,6 @@ proc suggestedFees*(chainId: int): RpcResponse[JsonNode] {.raises: [Exception].}
   let payload = %* [chainId]
   return core.callPrivateRPC("wallet_getSuggestedFees", payload)
 
-proc suggestedRoutes*(account: string, amount: string, token: string, disabledFromChainIDs, disabledToChainIDs, preferredChainIDs: seq[uint64], priority: int, sendType: int): RpcResponse[JsonNode] {.raises: [Exception].} =
-  let payload = %* [sendType, account, amount, token, disabledFromChainIDs, disabledToChainIDs, preferredChainIDs, priority]
+proc suggestedRoutes*(account: string, amount: string, token: string, disabledFromChainIDs, disabledToChainIDs, preferredChainIDs: seq[uint64], sendType: int): RpcResponse[JsonNode] {.raises: [Exception].} =
```
Comment (caybro) [inline src/backend/eth.nim:31]: Why is the `amount` a string?

File: src/app_service/service/transaction/service.nim [hunk 4]
```diff
@@ -387,7 +379,7 @@ QtObject:
   proc suggestedRoutesReady*(self: Service, suggestedRoutes: string) {.slot.} =
     self.events.emit(SIGNAL_SUGGESTED_ROUTES_READY, SuggestedRoutesArgs(suggestedRoutes: suggestedRoutes))
 
-  proc suggestedRoutes*(self: Service, account: string, amount: Uint256, token: string, disabledFromChainIDs, disabledToChainIDs, preferredChainIDs: seq[uint64], priority: int, sendType: int): SuggestedRoutesDto =
+  proc suggestedRoutes*(self: Service, account: string, amount: Uint256, token: string, disabledFromChainIDs, disabledToChainIDs, preferredChainIDs: seq[uint64], sendType: int): SuggestedRoutesDto =
```
Comment (caybro) [inline src/app_service/service/transaction/service.nim:382]: Why is the `amount` some kind of `int` and elsewhere a string?

**Review by caybro — COMMENTED** [review]
Got some questions around the type mismatches

**Review by saledjenic — APPROVED** [review]

**Review by jrainville — COMMENTED** [review]
Looks good. Just waiting to see how you address Lukas' comments.

File: src/backend/eth.nim (hunk 3 above)
Comment (alaibe) [reply src/backend/eth.nim:31]: The int amount are transformed to their hex value as string.
We don' t really use simple int but Uint256 as we deal with very big values.

At this stage it is a string as for the transport layer we use hex

**Review by alaibe — APPROVED** [review]

File: src/backend/eth.nim (hunk 3 above)
Comment (caybro) [reply src/backend/eth.nim:31]: I see, thanks for the explanation.

File: ui/imports/shared/popups/SendModal.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/popups/SendModal.qml:508]: I think you can still do this if you first convert the string to a JS Number

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml:508]: Hey what I need to display is this 

image


seems like toLocaleCurrencyString provides a result like -` usd 123`

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml:508]: @caybro can you please have a look at this too? I dont think the suggestion you made will help here.

Comment (caybro) [reply ui/imports/shared/popups/SendModal.qml:508]: Ah you mean you want a different order of {currency, symbol}?

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml:508]: @caybro  yes and not the symbol as in "$" but "USD"

Comment (caybro) [reply ui/imports/shared/popups/SendModal.qml:508]: I see, disregard pls :)

**Review by caybro — APPROVED** [review]
LGTM from my pov

**Pull request merged**